contracts/Vault.vy

```diff
@@ -1228,10 +1245,10 @@ def report(_return: uint256) -> uint256:
     # the credit increase we are offering (if any)
     # NOTE: This is just used to adjust the balance of tokens between the Strategy and
     #       the Vault based on the Strategy's debt limit (as well as the Vault's).
-    if _return < credit:  # credit surplus, give to Strategy
-        assert self.token.transfer(msg.sender, credit - _return)
-    elif _return > credit:  # credit deficit, take from Strategy
-        assert self.token.transferFrom(msg.sender, self, _return - credit)
+    if _gain < credit:  # credit surplus, give to Strategy
+        assert self.token.transfer(msg.sender, credit - _gain)
```

Comment:
`_gain` could be zero, so we would give back completed credit amount right?

Reply:
Yes, that's the idea